README.md

```diff
@@ -762,7 +758,7 @@ startup’s growth and how that affects who owns the stock.
   small successes (1X to 5X returns), **one in 8** are big successes (5X to 30x), and **one in
   20** are huge successes (30X+).
   - 🚧 What are some stats beyond angel investments?
-- 🔸 Each stage reflects the reduction of risk and increased dilution.
```

Comment:
Why this edit? Minor but no longer quite as logical as risk reduction is the primary goal.

Reply:
In this context the definite article implies a direct relationship b/n the two, b/c it's not referring to anything we've heard before (nothing phrased "the reduction of risk"). Yes the reduction of risk is primary, but if that's the concern you actually can end with the more important of the two for greater impact.